pulsar-flink-connector/src/main/java/org/apache/flink/streaming/connectors/pulsar/FlinkPulsarSink.java

```diff
@@ -51,9 +162,14 @@ public FlinkPulsarSink(
             PulsarSerializationSchema serializationSchema,
             MessageRouter messageRouter,
             PulsarSinkSemantic semantic) {
-
-        super(adminUrl, defaultTopicName, clientConf, properties, serializationSchema, messageRouter, semantic);
-        this.serializationSchema = serializationSchema;
+        this(new Builder<T>()
```

Comment:
The goal was basically to delegate as much as possible to a single c'tor that contains the interesting logic, instead of repeating variants of the logic inside the c'tor all over the place. 
But to be honest, I don't mind very much. So I'll just revert to the old c'tor.

Reply:
Back to the old c'tor.